zib/vocabulary/manual/ConceptMap-AdresSoortCodelijst-to-AddressType.xml

```diff
@@ -55,21 +55,21 @@
             <code value="TMP"/>
             <target>
                 <equivalence value="unmatched"/>
-                <comment value="Temporary Address/Tijdelijk adres. Use Address.use = 'temp'."/>
+                <comment value="TMP: Temporary Address/Tijdelijk adres. Use Address.use = 'temp'."/>
```

Comment:
Not sure if this is helpful. The code is already there. This works for these kinds of readable codes, but when it is some long SNOMED code it doesn't improve readability I guess.

Reply:
Agreed; I've removed the codes themselves at the start of the `comment`.